src/mame/mame.lst

```diff
@@ -41478,6 +41478,7 @@ gulfwar2a                       //
 hishouza                        // B02 / TP-007 (c) 1987 Taito Corporation (Japan)
 hishouzab                       // bootleg
 ktiger                          // B30 / TP-011 (c) 1987 Taito Corporation (Japan)
+skysharbl                       // bootleg
 skyshark                        // B02 / TP-007 (c) 1987 Taito America Corporation + Romstar license (US)
 skysharka                       // B02 / TP-007 (c) 1987 Taito America Corporation + Romstar license (US)
```

Comment:
This isn’t in order.

Reply:
Sorry about the question, but... It isn't?

Reply:
If you didn't remove the "k" then I suppose ;)

I'd keep the "k". By the way, IMO the setnames for the fshark bootlegs in this driver are a mess:
fsharkb, fsharkbt, fnshark

Reply:
I've renamed the set keeping the "k".

Thx!